src/boards/include/boards/waveshare_rp2350_touch_lcd_1.28.h

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2020 Raspberry Pi (Trading) Ltd.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+// -----------------------------------------------------
+// NOTE: THIS HEADER IS ALSO INCLUDED BY ASSEMBLER SO
+//       SHOULD ONLY CONSIST OF PREPROCESSOR DIRECTIVES
+// -----------------------------------------------------
+
+// pico_cmake_set PICO_PLATFORM=rp2350
+
+#ifndef _BOARDS_WAVESHARE_RP2350_TOUCH_LCD_1_28_H
+#define _BOARDS_WAVESHARE_RP2350_TOUCH_LCD_1_28_H
+
+// For board detection
+#define WAVESHARE_RP2350_TOUCH_LCD_1_28
+
+// --- RP2350 VARIANT ---
+#define PICO_RP2350A 1
+
+// --- UART ---
+#ifndef PICO_DEFAULT_UART
+#define PICO_DEFAULT_UART 0
+#endif
+#ifndef PICO_DEFAULT_UART_TX_PIN
+#define PICO_DEFAULT_UART_TX_PIN 0
+#endif
+#ifndef PICO_DEFAULT_UART_RX_PIN
+#define PICO_DEFAULT_UART_RX_PIN 1
+#endif
+
+// no PICO_DEFAULT_WS2812_PIN
+
+// --- I2C ---
+#ifndef PICO_DEFAULT_I2C
+#define PICO_DEFAULT_I2C 1
+#endif
+#ifndef PICO_DEFAULT_I2C_SDA_PIN
+#define PICO_DEFAULT_I2C_SDA_PIN 6
+#endif
+#ifndef PICO_DEFAULT_I2C_SCL_PIN
+#define PICO_DEFAULT_I2C_SCL_PIN 7
+#endif
+
+// --- SPI ---
+#ifndef PICO_DEFAULT_SPI
+#define PICO_DEFAULT_SPI 0
+#endif
+#ifndef PICO_DEFAULT_SPI_SCK_PIN
+#define PICO_DEFAULT_SPI_SCK_PIN 18
+#endif
+#ifndef PICO_DEFAULT_SPI_TX_PIN
+#define PICO_DEFAULT_SPI_TX_PIN 19
+#endif
+#ifndef PICO_DEFAULT_SPI_RX_PIN
+#define PICO_DEFAULT_SPI_RX_PIN 16
+#endif
+#ifndef PICO_DEFAULT_SPI_CSN_PIN
+#define PICO_DEFAULT_SPI_CSN_PIN 17
+#endif
+
+// --- LCD ---
+#ifndef WAVESHARE_LCD_SPI
+#define WAVESHARE_LCD_SPI 1
+#endif
+#ifndef WAVESHARE_LCD_DC_PIN
+#define WAVESHARE_LCD_DC_PIN 8
+#endif
+#ifndef WAVESHARE_LCD_CS_PIN
+#define WAVESHARE_LCD_CS_PIN 9
+#endif
+#ifndef WAVESHARE_LCD_SCLK_PIN
+#define WAVESHARE_LCD_SCLK_PIN 10
+#endif
+#ifndef WAVESHARE_LCD_TX_PIN
+#define WAVESHARE_TX_PIN 11
+#endif
```

Comment:
Slight typo here - presumably `WAVESHARE_TX_PIN` was supposed to be `WAVESHARE_LCD_TX_PIN` ?

Reply:
please forgive my colleague
For some reasons, he had to use the command line editor, which he was not used to, which led to him often making mistakes.
hope you understand

Reply:
No apologies needed!